CHANGELOG.md

```diff
@@ -7,7 +7,31 @@
 * None
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
+* Fixed enums which was accidentally exported on the `Realm` namespace without a backing implementation. ([#5493](https://github.com/realm/realm-js/pull/5493), since v11.0.0)
+* Converting local Realm to synced Realm crashes if an embedded object is null. ([#5389](https://github.com/realm/realm-js/issues/5389), since v10.13.0)
+* Fixed performance degradation on subqueries. ([realm/realm-core#6327](https://github.com/realm/realm-core/issues/6327), since v6.0.0)
+* Fixed crash if secure transport returns an error with a non-zero length. It can be observed as `Reading failed: Premature end of input` in the log. ([realm/realm-core#5435](https://github.com/realm/realm-core/issues/5435), since v10.0.0)
```

Comment:
"if secure transport returns an error with a non-zero length" - this sounds very internal?